frontend/src/AlertsManagement/management_helpers.js

```diff
@@ -45,7 +43,11 @@ export async function getAlertsData(alertsLimit) {
     }
   });
 
-  return [allAlerts, unresolvedAlerts, resolvedAlerts];
+  return {
+    all: allAlerts, 
+    unresolved: unresolvedAlerts,
+    resolved: resolvedAlerts
```

Comment:
also as an FYI -- probably out of scope of this PR (and internship) but i just caught this just now. is there a need to return the 'all' (duplicated from the set union of unresolved and resolved) given that the `all` can be derived from the unresolved+alert. 

this probably depends on the scale of how many alerts are returned but just wanted to put out my thoughts.

Reply:
(still thanks for the fyi lol) the allAlerts is a Map! I guess unresolved and resolved could be derived from all though, in which case I should probably not do the processing for that in this function. I guess when I was initially coding this up the function I had in mind would return all 3 because that's necessary for the AlertsContent component, but now that I have the History component that also fetches alerts then I shouldn't figure out which alerts are unresolved and resolved in this function.